### PR TITLE
Add command to close views

### LINKED ISSUE #216: Add "special" rootston command to close the focused window
Special meaning similar to the quit command.

### PR DESCRIPTION
**Pull request by emersion**
Fixes #216

### CODE REVIEW

**Comment by emersion** [comment]
wlc [destroys the `wl_client`](https://github.com/Cloudef/wlc/blob/704c56bf66f87f8788ab352a60d4ad0108b1920c/src/compositor/view.c#L503) if the surface implements wl_shell. Should we do that? Should we close `wl_client` for surfaces that aren't wl_shell too?

**Comment by SirCmpwn** [comment]
That seems reasonable to me. wl_shell clients are going to become rarer and rarer anyway.

File: rootston/rootston.ini.example [hunk 1]
```diff
@@ -30,6 +30,7 @@ meta-key = Logo
 
 # Keybindings
 # Maps key combinations with commands to execute
-# The special command "exit" stops the compositor
+# Use the prefix "exec" to execute a shell command
 [bindings]
-Logo+q = exit
+Logo+q = exit # Stops the compositor
+Logo+w = close # Closes the current view
```
Comment (SirCmpwn) [inline rootston/rootston.ini.example]: Just for fun let's match the default sway keybindings. mod+q closes the current view and mod+shift+e exits sway.

**Comment by SirCmpwn** [comment]
LGTM

**Comment by SirCmpwn** [comment]
Thanks!

**Pull request merged**